Buildings/Examples/VAVReheat/Controls/Economizer.mo

```diff
@@ -2,264 +2,242 @@ within Buildings.Examples.VAVReheat.Controls;
+            {240,100}})));
   Modelica.Blocks.Math.Gain gain(k=1/VOut_flow_min) "Normalize mass flow rate"
-    annotation (Placement(transformation(extent={{-60,-60},{-40,-40}})));
+    annotation (Placement(transformation(extent={{-60,-70},{-40,-50}})));
   Buildings.Controls.Continuous.LimPID conV_flow(
     k=k,
     Ti=Ti,
     yMax=0.995,
     yMin=0.005,
     Td=60,
+    y_reset=0,
     final reset=if have_reset then Buildings.Types.Reset.Parameter else
       Buildings.Types.Reset.Disabled)
     "Controller for outside air flow rate"
-    annotation (Placement(transformation(extent={{-22,-20},{-2,0}})));
+    annotation (Placement(transformation(extent={{-10,-30},{10,-10}})));
   Modelica.Blocks.Sources.Constant uni(k=1) "Unity signal"
-    annotation (Placement(transformation(extent={{-60,-20},{-40,0}})));
-  parameter Real k=1 "Gain of controller";
-  parameter Modelica.SIunits.Time Ti "Time constant of integrator block";
-  Modelica.Blocks.Interfaces.RealOutput yOA
-    "Control signal for outside air damper" annotation (Placement(
-        transformation(extent={{200,60},{240,100}}),iconTransformation(extent={{200,60},
-            {240,100}})));
-  Modelica.Blocks.Routing.Extractor extractor(nin=6, index(start=1, fixed=true))
-    "Extractor for control signal"
-    annotation (Placement(transformation(extent={{120,-10},{140,10}})));
+    annotation (Placement(transformation(extent={{-60,-30},{-40,-10}})));
   Modelica.Blocks.Sources.Constant closed(k=0) "Signal to close OA damper"
-    annotation (Placement(transformation(extent={{60,-90},{80,-70}})));
+    annotation (Placement(transformation(extent={{30,30},{50,50}})));
   Modelica.Blocks.Math.Max max
     "Takes bigger signal (OA damper opens for temp. control or for minimum outside air)"
     annotation (Placement(transformation(extent={{80,-10},{100,10}})));
-  MixedAirTemperatureSetpoint TSetMix "Mixed air temperature setpoint"
-    annotation (Placement(transformation(extent={{-20,64},{0,84}})));
-  EconomizerTemperatureControl yOATMix(
-    final Ti=Ti,
-    final k=k,
-    final have_reset=have_reset)
-    "Control signal for outdoor damper to track mixed air temperature setpoint"
-    annotation (Placement(transformation(extent={{20,160},{40,180}})));
   Buildings.Controls.Continuous.LimPID yOATFre(
     k=k,
     Ti=Ti,
     Td=60,
     yMax=1,
     yMin=0,
+    y_reset=1,
     reverseActing=false,
     final reset=if have_reset then Buildings.Types.Reset.Parameter else
-      Buildings.Types.Reset.Disabled)
+      Buildings.Types.Reset.Disabled) if have_frePro
     "Controller of outdoor damper to track freeze temperature setpoint"
-    annotation (Placement(transformation(extent={{20,120},{40,140}})));
+    annotation (Placement(transformation(extent={{-30,70},{-10,90}})));
   Modelica.Blocks.Math.Min min
     "Takes bigger signal (OA damper opens for temp. control or for minimum outside air)"
```

Comment:
Correct the comment `Takes smaller signal (OA damper opens for freeze protection control or for minimum outside air.)`

Reply:
Addressed